src/steps/opportunity/opportunity-delete.ts

```diff
@@ -15,13 +15,24 @@ export class DeleteOpportunity extends BaseStep implements StepInterface {
     type: FieldDefinition.Type.ANYSCALAR,
     description: 'the value of the field',
   }];
+  protected expectedRecords: ExpectedRecord[] = [{
+    id: 'opportunity',
+    type: RecordDefinition.Type.KEYVALUE,
+    fields: [{
+      field: 'Id',
+      type: FieldDefinition.Type.STRING,
+      description: "Opportunity's SalesForce ID",
+    }],
+    dynamicFields: true,
+  }];
 
   async executeStep(step: Step): Promise<RunStepResponse> {
     const stepData: any = step.getData().toJavaScript();
 
     try {
       const result = await this.client.deleteOpportunityByIdentifier(stepData.field, stepData.identifier);
-      return this.pass('Successfully deleted Opportunity with %s %s', [stepData.field, stepData.identifier]);
+      const record = this.keyValue('opportunity', 'Deleted Opportunity', { Id: result.id });
```

Comment:
- [ ] There may be a regression here, because, while the step does appear to actually delete the opportunity, the step appears as though it fails due to possibly passing `undefined` from the Cog to Crank.

![Screen Shot 2020-02-25 at 1 50 22 PM](https://user-images.githubusercontent.com/3496491/75290744-c9972d80-57d5-11ea-800b-3339045067ff.png)